### PR TITLE
Wanderers no longer offer amenities before translator

### LINKED ISSUE #666: Can visit shipyard, hire crew, trade with wanderers before being able to communicate?
It seems to me that all of these features should be disabled upon landing on the first Wanderer world, currently they are all available.

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Bug fix**

Fixes #666.

## Summary
The Wanderers will no longer do anything for you (besides show news) until you bring the Hai translator.

## Additional Context
With this PR the Wanderers don't yet refuel you, though MZ said maybe they should [here](https://github.com/endless-sky/endless-sky/issues/666#issuecomment-172218021).

## Screenshots
None

## Testing Done
Works for me.

## Save File
This save file can be used to test these changes:
[Drin Brock~3018-09-26-pre-wanderer-first-contact.txt](https://github.com/user-attachments/files/17415069/Drin.Brock.3018-09-26-pre-wanderer-first-contact.txt)

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
That is not the title I wanted to open it with, sorry.
Thoughts on letting the Wanderers refuel you?

File: data/map planets.txt [hunk 1]
```diff
@@ -2445,8 +2445,10 @@ planet "Kort Kehai"
...
-	spaceport `This is the only Wanderer spaceport on the entire planet, with a population of no more than a few hundred Wanderers, most of them engineers and robot technicians. At the center of the port is a glass dome nearly a hundred meters tall, enclosing a garden with cliffs, streams, and towering trees, where the Wanderers can stretch their wings and relax in a green and growing environment without being exposed to the outside air.`
-	spaceport `	The hallway that loops around the outside of the garden is decorated with an incredibly detailed mural. At one end the mural depicts a barren and blasted landscape, which gradually transitions along the length of the hallway to a green and growing world. The mural must be a reminder to the workers of what they are striving to accomplish here.`
+	port Spaceport
```
Comment (TomGoodIdea) [inline data/map planets.txt:2448]: If you don't specify the name, it should be "Spaceport" by default.

Comment (TheGiraffe3) [reply data/map planets.txt:2448]: Are you sure?

Comment (TheGiraffe3) [inline data/map planets.txt:2448]:
```suggestion
	port
```

File: data/map planets.txt [hunk 2]
```diff
@@ -2457,7 +2459,9 @@ planet "Kort Vek'kri"
 	attributes wanderer
 	landscape land/mountain10-sfiera
 	description `This world is nearly unpopulated, with very little animal life and strange, twisted plants with odd coloration. In small patches of land, the native vegetation gives way to green forests and fields, which were probably planted there by the Wanderers. In some of the larger forests, birds and insects have begun to flourish.`
-	spaceport `This small spaceport is unmistakably a research station, and the Wanderers walking or flying from one building to another have a self-absorbed attitude not unlike what you would expect from human scientists. Around the outskirts of the port village are pens that hold a staggering variety of animals, mostly ruminants and other herbivores, and the work of the scientists seems to revolve around convincing these animals to eat the local flora.`
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:2462]:
```suggestion
	port
```

File: data/map planets.txt [hunk 3]
```diff
@@ -4501,8 +4505,10 @@ planet "Tik Klai"
 	attributes research wanderer
 	landscape land/badlands2
 	description `This world is uncomfortably hot, even for the Wanderers, and so close to its star that the surface is bathed in dangerous levels of ultraviolet light. The few Wanderer installations here are buried underground, with only a few access hatches and storage buildings on the surface.`
-	spaceport `Most of the spaceport facility is closed to non-Wanderer personnel, and apparently to many of the off-world Wanderers who are visiting, as well. The underground passageways are comfortably wide and tall for a human, but the Wanderers walking through them must tuck their wings in close to their bodies to avoid scraping against the ceilings.`
-	spaceport `	The warning signs marking restricted areas are in the undecipherable Wanderer language, but accompanied by a symbol you easily recognize: a DNA double helix undergoing replication.`
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4508]:
```suggestion
	port
```

File: data/map planets.txt [hunk 4]
```diff
@@ -4691,10 +4697,10 @@ planet "Var' Kar'i'i"
...
 	description `This is a populous Wanderer world, with small farming villages scattered across all the land, and even a few larger population centers that might almost qualify as a city. But from the air, it is not evident at first that the planet is so heavily settled, because the village tree-houses are mostly hidden beneath the forest canopies. The farms, too, are almost indistinguishable from natural land, due to a permaculture system where many varieties of plants are growing side by side.`
-	spaceport `Apparently, just as humans use horses to carry loads and draw carts of produce, the Wanderers use a large breed of lizard, nearly three meters tall and walking on their hind legs. The lizards are meat-eaters: next to the watering troughs are cages full of large rodents, which the Wanderers occasionally feed to their beasts.`
-	spaceport `	Many different forms of produce are for sale here in open-air markets, including dozens of different nuts and berries. Aside from the rodents to feed to the lizards, and a few species of fish, the Wanderers do not seem to raise animals for food.`
-	shipyard "Wanderer Basics"
-	outfitter "Wanderer Basics"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4700]:
```suggestion
	port
```

File: data/map planets.txt [hunk 5]
```diff
@@ -4705,10 +4711,10 @@ planet "Var' Kayi"
 	attributes urban wanderer
 	landscape land/fields13
 	description `This is an Earth-like world with one major Wanderer city and countless smaller settlements. The settlements mostly consist of tree-houses built high up in living trees. The architecture of the city skyscrapers is full of cantilevered balconies and arches, mimicking the natural shape of a forest. Here and there a bit of smoke or steam rises from what must be a factory or power plant, but most of the land outside the city is green fields and pristine oceans.`
-	spaceport `This spaceport is as busy and chaotic as any major human trade hub, with the added complication that many of the Wanderers are flying from building to building instead of walking. Most of the buildings have balconies on every level, allowing visitors to enter from the air as well as from the streets.`
-	spaceport `	The starship landing pads are well outside the city, and ships land and take off along very tightly controlled vectors, probably to avoid midair collisions between the ships and the Wanderers who are flying on their own wings.`
-	shipyard "Wanderer Advanced"
-	outfitter "Wanderer Advanced"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4714]:
```suggestion
	port
```

File: data/map planets.txt [hunk 6]
```diff
@@ -4719,7 +4725,9 @@ planet "Var' Roi"
 	attributes frozen moon research wanderer
 	landscape land/snow14
 	description `The surface of this small icy moon is constantly cracked and broken by tidal force from the gas giant that it orbits. The tidal disturbances also heat the moon enough to keep its oceans warm enough to support life despite the small amount of sunlight that reaches here.`
-	spaceport `The only habitation on this moon is a Wanderer research station suspended on metal pylons well above the icy surface of the ocean. The gravity here is low enough that even though the atmosphere is thin, the Wanderers who work in the station can fly out across the ocean on their own power, wearing some sort of breathing gear. The researchers also have a fleet of submarine ships, sturdy enough to pierce through the shifting ice and travel deep below the ocean surface.`
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4728]:
```suggestion
	port
```

File: data/map planets.txt [hunk 7]
```diff
@@ -4731,9 +4739,9 @@ planet "Vara K'chrai"
...
 	description `	There are also farms here, but more like gardens or parks than a human farm: each with dozens of species of plants growing side by side, and harvested by autonomous robots.`
-	spaceport `The streets in the Wanderer capital city seem to be mostly reserved for cargo vehicles; the Wanderers themselves fly from one building to another instead, and the skyscrapers have balconies dozens of stories above the ground where visitors can land or take off. The air traffic seems utterly chaotic, but somehow they avoid colliding with each other in midair, and certain columns of airspace have been reserved for ships taking off and landing so that flying Wanderers will not get caught in the backwash from the ship engines.`
-	shipyard "Wanderer Advanced"
-	outfitter "Wanderer Advanced"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4742]:
```suggestion
	port
```

File: data/map planets.txt [hunk 8]
```diff
@@ -4743,7 +4751,9 @@ planet "Vara Ke'sok"
 	attributes fishing wanderer
 	landscape land/beach6
 	description `The surface of this world is almost entirely ocean, and the Wanderers who inhabit it live on massive floating algae mats, some of them the size of a small city. Engines are attached to some of these floating villages, allowing them to be slowly propelled from one part of the planet's surface to another, and the only native industries are fishing and seaweed farming.`
-	spaceport `The raft of algae that supports the spaceport is probably at least a dozen meters thick, but flexible enough that it bends as the ocean swells pass underneath, causing individual buildings to rise up or tilt slightly relative to their neighbors. The effect is subtle, but disconcerting, and you cannot help but stumble drunkenly at times as you explore the island.`
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4754]:
```suggestion
	port
```

File: data/map planets.txt [hunk 9]
```diff
@@ -4754,8 +4764,9 @@ planet "Vara Ke'stai"
 	attributes farming wanderer
 	landscape land/desert11
 	description `This is a desert world, but the Wanderers are working little by little to make the desert bloom: planting hardy shrubs and cacti that eventually transform the sand into a soil that can hold moisture. In some regions, especially closer to the poles, the deserts have given way to dense forests, but near the equator there are still large stretches of dry and barren sand, broken by the occasional belt of green where a narrow river winds between the dunes.`
-	spaceport `The spaceport is in a forest village near the planet's north pole. The landing pads are massive stone monoliths laid on the ground in a nearby clearing. But aside from a few large warehouses at ground level, the village itself is entirely made of tree-houses. A few mechanical lifts have been built into the sides of the trees to carry cargo up to the forest canopy where the Wanderers live, but the Wanderers themselves seem content to fly between the houses rather than using the lifts.`
-	outfitter "Wanderer Basics"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4767]:
```suggestion
	port
```

File: data/map planets.txt [hunk 10]
```diff
@@ -4766,8 +4777,9 @@ planet "Vara Kehi'ki"
 	attributes farming research wanderer
 	landscape land/fields14
 	description `This is a farming world, but the primary crops are medicinal plants rather than food: fields of brightly colored flowers and delicate shrubs. Large factories near the farms process the raw plants and extract useful chemical compounds. Their gleaming metal smokestacks belching steam into the air seem incongruous amid the natural beauty and bright colors of the fields of flowers.`
-	spaceport `The spaceport village is apparently not just a processing facility for the medicinal plants, but also a hospital of sorts. The village is designed to allow easy foot traffic as well as flight between buildings, and thousands of Wanderers, many of them appearing sick or elderly, live here. The main streets of the village are lined with vine trellises with flowers of every color imaginable, and mechanical lifts allow even the most frail Wanderers to travel up to the higher terraces and balconies.`
-	outfitter "Wanderer Basics"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4780]:
```suggestion
	port
```

File: data/map planets.txt [hunk 11]
```diff
@@ -4788,9 +4800,9 @@ planet "Vara Rakak"
 	attributes farming wanderer
 	landscape land/forest1
 	description `This is one of those rare worlds whose climate is already nearly perfect without the need for terraforming. The Wanderers have made it into a farming planet. The fields follow the natural contours and geology of the land rather than being laid out in rectangular lots like most human farms, and each field contains several different crop species designed to form a symbiotic ecosystem.`
-	spaceport `This could be a spaceport on any human farming world, except that the farmers hawking their wares have wings and beaks, and the wagons bringing produce to the market are drawn by massive lizards instead of horses. In several repair shops on the outskirts of town, Wanderer technicians work on massive harvester robots. The robots have four legs to walk on and dozens of dexterous manipulator arms for picking crops, and each arm has a camera eye on it.`
-	shipyard "Wanderer Basics"
-	outfitter "Wanderer Advanced"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4803]:
```suggestion
	port
```

File: data/map planets.txt [hunk 12]
```diff
@@ -4801,9 +4813,10 @@ planet "Varu Ek'lak'lai"
 	attributes mining wanderer
 	landscape land/canyon7
 	description `The Wanderers harvest metal ores not from planets, but from asteroids, in order to not expend even more of the resources of the planets that are under their care. This star system contains an unusually high concentration of metallic asteroids, and as a result they have built many large refineries here to process the ore that their drones harvest.`
-	spaceport `The vast majority of ships landing and taking off from this port are unmanned Wanderer drones used for mining asteroids. The drones are maneuverable enough to avoid collisions with the few ships that land under manual control.`
-	spaceport `	The port itself is a small village surrounded by towering refineries. Unlike other Wanderer architecture, the refineries are ugly, brutal buildings made of weathered concrete.`
-	outfitter "Wanderer Basics"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4816]:
```suggestion
	port
```

File: data/map planets.txt [hunk 13]
```diff
@@ -4814,9 +4827,9 @@ planet "Varu K'est"
 	attributes military wanderer
 	landscape land/canyon5
 	description `Varu K'est is a spare, ancient planet of arid plateaus and deep, meandering canyons, with too little rainfall to support any but the hardiest of shrubs. The Wanderers have converted nearly all their villages on this planet into military bases in order to oppose the encroaching Hai raiders, and have also been forced to postpone their attempts at improving the local ecology.`
-	spaceport `The spaceport is a massive shipyard, with facilities for repairing the few warships that the Wanderers possess as well as creating new ships and weapons. Dozens of battle-scarred ships are parked in the hangars here, a stark contrast to the ports in the more peaceful regions of the Wanderer territory.`
-	shipyard "Wanderer Basics"
-	outfitter "Wanderer Advanced"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4830]:
```suggestion
	port
```

File: data/map planets.txt [hunk 14]
```diff
@@ -4827,8 +4840,9 @@ planet "Varu K'prai"
 	attributes oil wanderer
 	landscape land/sea4
 	description `This is one of the only Wanderer worlds that still has large reserves of oil, albeit deep below the ocean surface where mining is difficult. Aside from the drilling companies and refineries, the largest local employer is the local government's safety oversight department, whose inspectors are constantly visiting the drilling platforms and shipping hubs to ensure that all the regulations for preventing oil spills are being followed.`
-	spaceport `Plastics and other refined oil products are as essential to Wanderer technology as to humans, but it is clear that those responsible for harvesting them are not held in high regard. The crews of visiting ships seem eager to interact with the locals as little as possible. However, that does not appear to stop the Wanderer freighters from purchasing large shipments of plastic, which will no doubt yield a substantial profit elsewhere in Wanderer space.`
-	outfitter "Wanderer Basics"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4843]:
```suggestion
	port
```

File: data/map planets.txt [hunk 15]
```diff
@@ -4840,9 +4854,9 @@ planet "Varu Mer'ek"
 	attributes factory wanderer
 	landscape land/valley11-harro
 	description `This Wanderer factory world is now home to more abandoned villages than populated ones: entire factories rusting and decaying as a result of the recent threat from the Unfettered Hai. Most of the factories that remain active have been repurposed to produce weapons or ammunition for the defense of Wanderer space.`
-	spaceport `Very few Wanderer children are present in the spaceport. Most of the inhabitants still live far above the ground in the customary Wanderer tree-houses, but that portion of the village is dwarfed by the more recently constructed factories and warehouses for war supplies. A steady stream of freighters brings in new equipment and raw materials for the factories.`
-	shipyard "Wanderer Advanced"
-	outfitter "Wanderer Advanced"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4857]:
```suggestion
	port
```

File: data/map planets.txt [hunk 16]
```diff
@@ -4853,8 +4867,9 @@ planet "Varu Tek'kai"
 	attributes factory moon wanderer
 	landscape land/badlands4
 	description `This Wanderer world is almost uninhabited and devoid of any indigenous ecosystem, except perhaps on the microscopic level. It has none of the beauty of other Wanderer settlements. Here, factories have been built to process the most dangerous and destructive of industrial chemicals. If disaster strikes one of the factories, the only result will be the contamination of a world that is already dead and lifeless - far better, in the judgment of the Wanderers, than allowing such risky industries to operate on a living world.`
-	spaceport `Nearly all the operations of the factories here are automated, and the outside atmosphere is too thin for even the Wanderers to breathe. The workers live in dormitory apartments in a ring surrounding a large central dome. Inside the dome, protected from the elements, is a park whose gardens contain everything from towering conifers to fields of wildflowers; a necessary place of sanctuary and rest for the Wanderers who must spend the rest of their time servicing machines in the bleak environment outside the spaceport.`
-	outfitter "Wanderer Basics"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4870]:
```suggestion
	port
```

File: data/map planets.txt [hunk 17]
```diff
@@ -4865,8 +4880,9 @@ planet "Varu Tev'kei"
 	attributes factory wanderer
 	landscape land/canyon04
 	description `This is a desert world, too hot and too close to its sun to support much animal or plant life. The Wanderers have covered large stretches of the desert with solar-powered factories: smelters that use the sun's reflected heat to melt ore, and fields of solar panels that support other industries.`
-	spaceport `It is easy to distinguish the local workers from the visitors, because the locals wrap their wings in thin white sheets of cloth rather than leaving them bare: a necessary protection for any Wanderer who chooses to fly outside of the protective shade of the spaceport. Many of the buildings here extend deep underground, to where the surrounding bedrock is cooler than the hot sands at the surface.`
-	outfitter "Wanderer Basics"
+	port Spaceport
```
Comment (TheGiraffe3) [inline data/map planets.txt:4883]:
```suggestion
	port
```

File: data/map planets.txt (hunk 1 above)
Comment (TomGoodIdea) [reply data/map planets.txt:2448]: That's what the wiki says.

**Comment by warp-core** [comment]
I take it pilots who already did the mission where these facilities are being made available aren't allowed to use them anymore?

**Comment by TheGiraffe3** [comment]
Yes, good point. Just fixed that.

**Comment by Saugia** [comment]
Please be patient when it comes to PRs or issues. Bringing up something is one thing, but commenting to ask the status of a PR, or asking for us if something is going to be implemented soon, comes off rather unpleasant after a few times. We get to things as we get to them, no need to comment on every PR and ask about when we get to it or if its planned - it pings people and it can get upsetting.

A general rule of thumb is if a PR has not been looked at for a month or more, we put a label on it for either waiting for OP or waiting for reviewer/developer, and we check these weekly. Some PRs wait longer if they're bigger. 

As for this PR, it'd imagine it's best it warp gets a chance to look at it again, or if someone else feels comfortable about it.

File: data/wanderer/wanderers start.txt [hunk 18]
```diff
@@ -465,6 +466,19 @@ mission "Wanderers: Outfit License Patch"
 
 
 
+# Compatibility patch for pilots that did the above missions before 0.10.9.
+mission "Wanderer Technology Patch"
```
Comment (warp-core) [inline data/wanderer/wanderers start.txt:470]: Given that this is about planetary facilities, I don't think "technology" is a good name.

**Review by warp-core — COMMENTED** [review]
What happens to worlds that have been captured by the Unfettered if the patch mission happens after that point in the story?

**Comment by Saugia** [comment]
> I think that is addressed now.

Rather than saying something's addressed, it helps more to say what specifically occurs now (after testing) so we know (at least from your example) that something has changed. That, and something I've said before is save files to test the edge cases go a long way to help us review these PRs.

This is based on an old issue, so there are some edge cases with both old and new saves before and after updates that need testing, so it takes a minute to create saves on our end to test every edge case.

**Comment by TheGiraffe3** [comment]
Tested, confirmed this works.

**Comment by TheGiraffe3** [comment]
Added a save.

**Comment by TheGiraffe3** [comment]
Could somebody remove the `needs save` label, as there is one now?

**Comment by TheGiraffe3** [comment]
@warp-core when you have time, could you re-review this?
Tested with the provided save, works for me.

File: data/wanderer/wanderers start.txt [hunk 19]
```diff
@@ -468,6 +469,20 @@ mission "Wanderers: Outfit License Patch"
 
 
 
+# Compatibility patch for pilots that did the above missions before 0.10.9.
```
Comment (Amazinite) [inline data/wanderer/wanderers start.txt:472]:
```suggestion
# Compatibility patch for pilots that did the above missions before 0.10.13.
```

File: data/wanderer/wanderers start.txt [hunk 20]
```diff
@@ -1609,7 +1624,6 @@ mission "Wanderers: Unfettered Invasion Patch"
 		fail
```
Comment (Amazinite) [inline data/wanderer/wanderers start.txt:1626]: Please don't eat the whitespace.
```suggestion


```

File: data/wanderer/wanderers start.txt [hunk 21]
```diff
@@ -1628,6 +1642,20 @@ mission "Wanderers Invaded 0"
 
 
 
+# Compatibility patch for pilots that did the above missions before 0.10.9.
```
Comment (Amazinite) [inline data/wanderer/wanderers start.txt:1645]:
```suggestion
# Compatibility patch for pilots that did the above missions before 0.10.13.
```

File: data/wanderer/wanderers start.txt [hunk 22]
```diff
@@ -2711,6 +2739,21 @@ mission "Wanderers Rek 2"
...
```
Comment (Amazinite) [inline data/wanderer/wanderers start.txt:2742]:
```suggestion
# Compatibility patch for pilots that did the above missions before 0.10.13.
```

File: data/wanderer/wanderers.txt [hunk 23]
```diff
@@ -696,6 +696,65 @@ phrase "hostile disabled wanderer"
 	word
 		"."
 
+event "wanderers: all facilities available"
```
Comment (Amazinite) [inline data/wanderer/wanderers.txt:699]: I think that instead of having three events separated in time that copy/paste many of the same planets, you should have three events separated by which planets they affect. All three events would be triggered for the first compatibility mission, two events for the second, and only one event for the third, removing the event that updates the captured planets at the respective compatibility mission that excludes them now.

**Comment by TheGiraffe3** [comment]
I have not added a line to the first contact about this, because the rest of the things a spaceport normally does will happen even without a spaceport if the player has the proper outfits installed. Is there a way to get around that?

**Comment by Amazinite** [comment]
> the rest of the things a spaceport normally does will happen even without a spaceport if the player has the proper outfits installed. Is there a way to get around that?

No, and that's expected behavior, even if there is no spaceport at all on the planet that the player lands on.

**Comment by TheGiraffe3** [comment]
Well, my point is that it would be strange to tell the player "the aliens don't repair your hull" if they already have hull repair.

**Review by Amazinite — COMMENTED** [review]
I'm wondering now if we should just keep things simple and only make this change to Vara K'chrai, since that's the only planet you can land on before these events fire anyway.

I also think it's a little annoying to have to re-paste the spaceport descriptions, which can be prone to error, especially if we change the description of one of these planets in the map but then forget to change it in the events. I think you should just be able to do this in the events:
```
event "X"
	planet "Y"
		add port
			recharges all
			services all
			news # Technically redundant with what you have as the defaults, so this could be dropped.
```
That should be synonymous with completely redefining the port as a spaceport.

**Comment by Amazinite** [comment]
> Well, my point is that it would be strange to tell the player "the aliens don't repair your hull" if they already have hull repair.

Well it can be the case that the Wanderers refuse to repair or refuel your ship and you have the means of repairing or refueling yourself.

File: data/map planets.txt [hunk 24]
```diff
@@ -6074,7 +6074,10 @@ planet "Vara K'chrai"
...
-	spaceport `The streets in the Wanderer capital city seem to be mostly reserved for cargo vehicles; the Wanderers themselves fly from one building to another instead, and the skyscrapers have balconies dozens of stories above the ground where visitors can land or take off. The air traffic seems utterly chaotic, but somehow they avoid colliding with each other in midair, and certain columns of airspace have been reserved for ships taking off and landing so that flying Wanderers will not get caught in the backwash from the ship engines.`
+	port
+		recharges fuel
+		news
+		description `The streets in the Wanderer capital city seem to be mostly reserved for cargo vehicles; the Wanderers themselves fly from one building to another instead, and the skyscrapers have balconies dozens of stories above the ground where visitors can land or take off. The air traffic seems utterly chaotic, but somehow they avoid colliding with each other in midair, and certain columns of airspace have been reserved for ships taking off and landing so that flying Wanderers will not get caught in the backwash from the ship engines.`
 	shipyard "Wanderer Advanced"
 	outfitter "Wanderer Advanced"
```
Comment (Amazinite) [inline data/map planets.txt:6082]:
```suggestion
```

File: data/wanderer/wanderers start.txt [hunk 25]
```diff
@@ -455,6 +456,16 @@ mission "Wanderers Conversation"
...
+event "wanderers: vara k'chrai facilities"
+	planet "Vara K'chrai"
+		add port
+			recharges all
+			services all
+		add shipyard "Wanderer Advanced"
+		add outfitter "Wanderer Advanced"
+
+
+
```
Comment (Amazinite) [inline data/wanderer/wanderers start.txt:468]: We still need a compatibility mission for this one.
```suggestion

# Compatibility patch for pilots that did the above missions before 0.10.13.
mission "Wanderers: Vara K'Chrai Facilities Patch"
	landing
	invisible
	to offer
		has "Wanderers Conversation: offered"
	on offer
		event "wanderers: vara k'chrai facilities"
		fail



```
